pkg/apis/admission/types.go

```diff
@@ -73,6 +73,10 @@ type AdmissionRequest struct {
 	// OldObject is the existing object. Only populated for UPDATE requests.
 	// +optional
 	OldObject runtime.Object
+	// DryRun indicates that modifications will definitely not be persisted for this request.
+	// Defaults to false.
+	// +optional
+	DryRun *bool
```

Comment:
We don't want to pass on the different sorts of dry run calls?

Reply:
I'm not sure, I'm not the one arguing that dryRun should be an array of strings...

Reply:
I think this was discussed on the wg-apply slack channel, and the conclusion was that giving the webhooks the array of strings would be exposing a lot of complexity to them, which they don't necessarily need. Because we don't want to allow the level of granularity to dry run with only some of the admission webhooks, anyway.